internal/cmd/base/base.go

```diff
@@ -220,30 +227,46 @@ func (c *Command) Client(opt ...Option) (*api.Client, error) {
 
 	switch {
 	case c.FlagRecoveryConfig != "":
-		wrapper, err := wrapper.GetWrapperFromPath(c.FlagRecoveryConfig, "recovery")
+		wrapper, cleanupFunc, err := wrapper.GetWrapperFromPath(
+			c.Context,
+			c.FlagRecoveryConfig,
+			"recovery",
+			configutil.WithPluginOptions(
+				pluginutil.WithPluginsMap(kms_plugin_assets.BuiltinKmsPlugins()),
+				pluginutil.WithPluginsFilesystem("boundary-plugin-kms-", kms_plugin_assets.FileSystem()),
+			),
+			// TODO: How would we want to expose this kind of log to users when
```

Comment:
Reviewers: Note that this here and a few other places is for recovery wrappers. In general since those are sourced as part of a separate command we don't have logging set up. So it's not clear what to do here. For the moment swallowing it is likely fine, as we would have swallowed it anyways if it was built-in -- this means swallowing go-plugin status info but there are other ways to verify if that is an issue (such as with `boundary dev`).